sct.py

```diff
@@ -1161,8 +1161,8 @@ def run_pytest(target, backend, config, logdir):
         sys.exit(1)
     return_code = pytest.main(['-s', '-v', f'--junit-xml={junit_file}', '-p', 'no:warnings', target])
     test_config = get_test_config()
-    # temp, until we'll have API for sending it
-    test_config.argus_client().submit_sct_logs([LogLink(log_name=junit_file.name, log_link=str(junit_file))])
+    with open(junit_file, "rt", encoding="utf-8") as junit_xml:
+        test_config.argus_client().sct_submit_junit_report(file_name=junit_file.name, raw_content=junit_xml.read())
```

Comment:
how big these files are?
What is the limit of payload?

Reply:
No limit, they are saved as blobs into a separate table. The files are pretty small, about 50kb or so